config_examples/arm/README.md

```diff
@@ -0,0 +1,40 @@
+This directory contains samples packer templates to get you started.
+
+* CentOS v7.1
+* Debian v8
+* OpenSuSE v13.2
+* Ubuntu v14.04-3 LTS
+
+All of these scripts require that you supply values for the following
+variables.
+
+* cid (Client ID)
+* cst (Client Secret)
+* tid (Client Tenant)
+* sid (Subscription ID)
+* rgn (Resource Group Name)
+* sa  (Storage Account)
+
+```batch
+packer build^
+  -var cid="593c4dc4-9cd7-49af-9fe0-1ea5055ac1e4"^
+  -var cst="GbzJfsfrVkqL/TLfZY8TXA=="^
+  -var sid="ce323e74-56fc-4bd6-aa18-83b6dc262748"^
+  -var tid="da3847b4-8e69-40bd-a2c2-41da6982c5e2"^
+  -var rgn="My Resource Group"^
+  -var sa="mystorageaccount"^
+  c:\packer\ubuntu_14.04.3-LTS.json
+```

```

Comment:
these values better not work... or you go and roll your secret right now... :wink: